src/libaktualizr/asn1/asn1_message.cc

```diff
@@ -29,7 +35,7 @@ int Asn1SocketWriteCallback(const void* buffer, size_t size, void* priv) {
 
   while (len > 0) {
     // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-pointer-arithmetic)
-    ssize_t written = send(*sock, b + pos, len, MSG_NOSIGNAL);
+    ssize_t written = send(*sock, b + pos, len, flags_no_signal);
```

Comment:
You can also ignore SIGPIPE like this. I don't know if that is needed.

    int val = 1;
    setsockopt(*sock, SOL_SOCKET, SO_NOSIGPIPE, (void*)&val, sizeof(val));